tests/network/test_services.py

```diff
@@ -105,7 +105,7 @@ def test_get_balance_main_failure(self):
             mock_network_api_main.get_balance(MAIN_ADDRESS_USED1)
 
     def test_get_transactions_main_equal(self):
-        results = [api.get_transactions(MAIN_ADDRESS_USED1) for api in network_api_main.list_of_apis]
+        results = [[*api.get_transactions(MAIN_ADDRESS_USED1)] for api in network_api_main.list_of_apis]
         assert all_items_common(results[:100])
```

Comment:
`results[:100]` is surely a bug in the existing test, since `results` items are the result of each API (currently 2 of them)